lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -21,31 +21,27 @@ def perform_work
 
           normalizer.with_normalized do |tmpdir|
             schema = Settings.geohydra.postgis.schema || 'druid'
-            # encoding =  # XXX: these are hardcoded encodings for certain druids -- these should be read from the metadata somewhere
-            #   case druid
-            #   when 'bt348dh6363', 'cc936tf6277'
-            #     'LATIN1'
-            #   else
-            #     'UTF-8'
-            #   end
 
             # sniff out shapefile from extraction
-            Dir.chdir(tmpdir)
-            shpfn = Dir.glob('*.shp').first
-            sqlfn = shpfn.gsub(/\.shp$/, '.sql')
-            errfn = 'shp2pgsql.err'
-            logger.debug "load-vector: #{bare_druid} is working on Shapefile: #{shpfn}"
+            shp_filename = Dir.glob("#{tmpdir}/*.shp").first
+            sql_filename = shp_filename.gsub(/\.shp$/, '.sql')
+            stderr_filename = "#{tmpdir}/shp2pgsql.err"
```

Comment:
`shp2pgsql` and `psql` seemed happy to run against more nested relative paths below the current directory, without changing to the directory where the files were being written/read.  at least afaict from unit and integration tests.